python/ml4ir/applications/ranking/model/layers/masking.py

```diff
@@ -160,8 +160,6 @@ def __init__(self,
         self.mask_at_inference = mask_at_inference
         self.requires_mask = requires_mask
 
-        super(QueryFeatureMask, self).__init__(**kwargs)
```

Comment:
@mohazahran you should still do `super().__init__(**kwargs)

Reply:
ping

Reply:
@lastmansleeping It's being called in a previous line. This was duplicate